pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -113,19 +114,24 @@ func gcObjects(ctx *volumemgrContext, dirName string) {
 		return
 	}
 	for _, location := range locations {
+		filelocation := path.Join(dirName, location.Name())
 		volumeName := strings.Split(location.Name(), ".")
 		if len(volumeName) > 1 {
```

Comment:
For clarity I would 
1. call this nameAndFormat instead of volumeName.
2. rephrase the "bad" log message to say "unknown format".
3. Reduce identation by having the unknown format code first, with a continue at the end.
4. Do you really need that Format_value code? You already have a string in upper case. What does other code do which parses the name of the volume? (and can that code be reused so we don't need this knowledge sprinkled across the code?)

Reply:
Implemented all 4 suggestions. Moved common code in a single function.